Simplification of the realtime profile, code de-duplication

yarda opened the pull request:
Signed-off-by: Jaroslav Škarvada jskarvad@redhat.com

yarda committed:
Simplification of the realtime profile, code de-duplication

Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

yarda committed:
realtime: added verification

Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

jeremyeder commented on profiles/realtime/variables.conf:
```diff
@@ -6,5 +6,6 @@
 #
 #
 
+isolated_cores = 1
```
I think this should be commented out.

yarda committed:
Do not preset isolated_cores

Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

yarda commented:
Removed in 960da9e177f8ce67933751a41f8ee52a53d24e2d

Commit:
guests should not use nohz_full=

That increases latencies for our (very specific) test-case because
when cyclictest goes to sleep the tick may get activated if other
tasks besides stress gets runnable. When cyclictest wakes up, the
tick is deactivated. If this activiate/deactivate cycle happens
way too often, latency increases because there's an overhead to
activiate/deactivate the tick.

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>

jeremyeder committed:
Revert "guests should not use nohz_full="

will revisit this later and may end up putting it back.

This reverts commit bb0404695d608c884a9235a38215aa9a337af66e.

yarda committed:
realtime: fixed / simplified profile according to comments

Signed-off-by: Jaroslav Škarvada <jskarvad@redhat.com>

yarda commented:
This pull request should be now up to date with the upstream tuned git and is ready for merge.

jeremyeder commented:
I see the tuna "-i" option in the updated script.sh, which was missing from the initial PR.  Thanks to acme for catching that.
